src/queries/products.ts

```diff
@@ -9,7 +9,7 @@ export function useAvailableProducts() {
     "available-products",
     async () => {
       const res = await axios.get<AvailableProduct[]>(
-        `${API_PATHS.bff}/product/available`
+        `${API_PATHS.bff}/available`
```

Comment:
This will request the data from `https://6me96m3are.execute-api.eu-west-1.amazonaws.com/dev//products/available` which you don't have in the API. 

That's why products are not showing up in the UI.

Reply:
I just fixed this route. Thanks.